_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -122,7 +123,7 @@ These two HTML `a` elements have the same [accessible name][] and [context][prog
 
 #### Passed Example 4
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context], and resolve to pages that serve the same purpose because the content section is the same.
+These two HTML `a` elements have the same [accessible name][], [accessible description][] and [context][programmatically determined link context], and resolve to pages that serve the same purpose because the content section is the same.
```

Comment:
May I that you replace this:
These two HTML `a` elements have the same [accessible name][], [accessible description][] and [context][programmatically determined link context], and resolve to pages that serve the same purpose because the content section is the same.
... with this:
These two HTML `a` elements have the same [accessible name][], the same [accessible description][], the same [context][programmatically determined link context], and resolve to pages that serve the same purpose because the content section is the same.

Maybe I'm being overly particular, but I suspect that small changes like this can make sentences a lot easier to parse in the mind of the reader.  Personally I am a fan of removing "implied parentheses" in a sentence where I can.  In this case the original version of the sentence has "implied parentheses" which apply the words "the same" to three things: 1) accessible name, 2) accessible description, and 3) context.  So I need to jump around in the sentence and in my mind.